src/main/resources/swagger/api-docs.yaml

```diff
@@ -3255,6 +3255,53 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/TermsOfServiceConfigResponse'
+
+  /termsOfService/v1/user/self:
+    get:
+      tags:
+        - TermsOfService
+      summary: Returns details about the requesting user's terms of service status.
+      operationId: userTermsOfServiceGetSelf
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UserTermsOfServiceDetails'
+        404:
+          description: user not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorReport'
+
+  /termsOfService/v1/user/{sam_user_id}:
+    get:
+      tags:
+        - TermsOfService
+      summary: gets a user's terms of service status, can only get another user's status if the requester is an admin.
+      operationId: userTermsOfServiceGet
+      parameters:
+        - name: sam_user_id
+          in: path
+          description: the id of the sam user to get terms of service details for
```

Comment:
I think we should use consistent language about what the object sent back is. I personally prefer status, since I dont think details really means anything here.